docs: Update for PT2 Extended Type Support (#190)
triton-inference-server/pytorch_backend: README.md

Reviewer: ```suggestion
`config.pbtxt`.

> [!IMPORTANT]
> Only applies with `platform: "pytorch_libtorch"`.
```

Author: fixed. thanks.

```bash
apt-get install rapidjson-dev python3-dev python3-pip
pip3 install patchelf==0.18.0
```

An appropriate PyTorch container from [NVIDIA NGC Catalog](https://ngc.nvidia.com) must be used.
For example, to build a backend that uses the 26.04 version of the PyTorch container from NGC:

```bash
mkdir build
cd build
cmake -DCMAKE_INSTALL_PREFIX:PATH=`pwd`/install -DTRITON_PYTORCH_DOCKER_IMAGE="nvcr.io/nvidia/pytorch:26.04-py3" ..

> make install

Reviewer: ```suggestion
make -j<N> install
```

Reviewer: Prevent user waiting forever to build

Author: fixed. thanks.

The full source for these PyTorch versions are available as Docker images from
[NGC](https://ngc.nvidia.com).

> For example, the PyTorch version compatible with the 26.02 release of Triton is available as `nvcr.io/nvidia/pytorch:26.05-py3` which supports PyTorch version ` 2.12.0a0`.

Reviewer: You mean 26.04?

Author: fixed. thanks.

> For example, the PyTorch version compatible with the 26.02 release of Triton is available as `nvcr.io/nvidia/pytorch:26.05-py3` which supports PyTorch version ` 2.12.0a0`.

Reviewer: ```suggestion
For example, the PyTorch version compatible with the 26.04 release of Triton is available as `nvcr.io/nvidia/pytorch:26.04-py3` which supports PyTorch version ` 2.12.0a0`.
```

Reviewer: Update or simply remove pytorch version.

Author: fixed. thanks.

#### Basic Rules for Forward Based I/O Names

* Addressing a tuple uses Pythonic tuple semantics such as `ARGS[0]`, `ARGS[0]`, and `RESULT[0]`.
* Addressing a list is identical to addressing a tuple.
* Addressing a dictionary uses semi-Pythonic dictionary semantics such as `RESULT[key]`, `KWARGS[key]`, and `ARGS[3][key]`.
* When the a model's output is a single tensor, it is addressed as `RESULT` with no suffix accessors.

#### Example

```python
import torch

SHAPE = (1, 16)

# Define a model with complex inputs and outputs.
# Include the eventual name mappings based on the return value of the model and sample inputs.
class ExampleAotiModel(torch.nn.Module):
  def __init__(self):
    super().__init__()

  def forward(
    self,
    hdata: torch.Tensor,
    vdata: torch.Tensor,
    options: dict[str, torch.Tensor],
    *
    kwag1: torch.Tensor,
    kw_arg2: torch.Tensor
  ) -> dict[
    str, torch.Tensor
    | tuple[torch.Tensor, torch.Tensor]
    | dict[str, torch.Tensor]
  ]:
    return {
      # OUTPUT__0 or RESULT[AAA]
      "AAA": hdata + vdata,
      # OUTPUT__1 or RESULT[ZZZ]
      "ZZZ": hdata - vdata,

>       "BBB": (

Reviewer: Why not alphabetical order?

Author: specifically to prove a point.

They'll listed in alphabetical order in the config.pbtxt, but their ordering does not align with their ordinal ordering.

Reviewer: Maybe add a comment to highlight this detail.

Author: done.

* Key: `NUM_THREADS`
* Value: The number of threads used for intra-op parallelism on CPU.

> > [!IMPORTANT]

Reviewer: Since all custom parameters has this restriction, why not move under Customization heading?

Author: Done.

}
```

> > [!IMPORTANT]
> > Only applies with `platform: "pytorch_libtorch"`.
> 
> ## Important Notes

Reviewer: ```suggestion
}
```

## Important Notes
```

Author: done.
